package.json

```
@@ -9,12 +9,15 @@
     "lint:json": "find public/**/v1/json/index.html -print0 | xargs -0I {} jsonlint '{}'",
     "lint:markdown": "markdownlint --config ./.markdown-lint.yml ./content/**/*.md --ignore './content/apis/*.md'",
     "lint:markdown:format": "markdownlint --fix --config ./.markdown-lint.yml ./content/**/*.md --ignore './content/apis/*.md'",
+    "lint:styles": "npx stylelint ./themes/**/*.scss --fix",
     "local-build": "gulp buildAssets && gulp watch",
     "start": "npm-run-all --parallel local-build hugo workflow",
     "prettier:templates": "npx prettier -w ./themes/**/*.html",
-    "test:templates": "npx prettier -c ./themes/**/*.html",
+    "prettier:styles": "npx prettier -w ./themes/**/*.scss",
     "test:htmlproofer": "htmlproofer ./public --allow-hash-href --check-html --empty-alt-ignore --disable-external",
     "test:pa11y": "npx pa11y-ci --config .pa11yci",
+    "test:styles": "npx prettier -c ./themes/**/*.scss",
+    "test:templates": "npx prettier -c ./themes/**/*.html",
```

Same with testing and fixing formatting.

```suggestion
    "prettier:styles": "npx prettier -c ./themes/**/*.scss",
    "prettier:styles:fix": "npx prettier -w ./themes/**/*.scss",
    "prettier:templates": "npx prettier -c ./themes/**/*.html", 
    "prettier:templates:fix": "npx prettier -w ./themes/**/*.html",
    "test:htmlproofer": "htmlproofer ./public --allow-hash-href --check-html --empty-alt-ignore --disable-external",
    "test:pa11y": "npx pa11y-ci --config .pa11yci",
```